src/config-import.js

```diff
@@ -36,23 +39,31 @@ const MAX_ICON_SIZE = 10_000_000
 export async function readConfig(configPath) {
   /** @type {Error[]} */
   const warnings = []
+  console.log('path', configPath)
 
   const zip = await yauzl.open(configPath)
   if (zip.entryCount > MAX_ENTRIES) {
     // MAX_ENTRIES in MAC can be inacurrate
     throw new Error(`Zip file contains too many entries. Max is ${MAX_ENTRIES}`)
   }
   const entries = await zip.readEntries(MAX_ENTRIES)
-  const [presetsFile, translationsFile] = await Promise.all([
-    findPresetsFile(entries),
-    findTranslationsFile(entries),
-  ])
+  const presetsFile = await findPresetsFile(entries)
+  const translationsFile = await findTranslationsFile(entries)
+  const metadataFile = await findMetadataFile(entries)
+  assert(
+    isValidConfigFile(metadataFile),
+    `invalid or missing config file version ${metadataFile.fileVersion}. We support version ${SUPPORTED_CONFIG_VERSION}}`
+  )
 
   return {
     get warnings() {
       return warnings
     },
 
+    get metadata() {
+      return { ...metadataFile, importDate: new Date().toISOString() }
```

Comment:
this will return a different value every time this property is accessed. The date should be set when import happens.